src/fpackdlg.pas

```diff
@@ -398,6 +411,8 @@ procedure TfrmPackDlg.SwitchOptions(ArcTypeChange: Boolean); // Ugly but working
           else
             sCmd:= FAdd;
 
+          lblPacker.Caption:='MultiArc [' + FPacker + ']';
```

Comment:
I don't like the `MultiArc` prefix, I think it will be better without it. We can distinguish what is used by square brackets and  `.wcx` extension.

Reply:
ok, i just added this label because of the issue when one extension is used several times, to see which packer will actually be used. 

btw, maybe instead of for loops, extensions should be stashed to some stringlist with userdata or something, to remove duplicates since dc doesnt use them anyway